### PR TITLE
openmw: 0.43.0 -> 0.44.0

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by matthewbauer** [comment]
@grahamcofborg build openmw

File: pkgs/games/openmw/default.nix [hunk 1]
```diff
@@ -11,14 +11,14 @@ let
...
-  version = "0.43.0";
+  version = "0.44.0";
   name = "openmw-${version}";
 
   src = fetchFromGitHub {
     owner = "OpenMW";
     repo = "openmw";
-    rev = name;
-    sha256 = "1nybxwp77qswjayf0g9xayp4x1xxq799681rhjlggch127r07ifi";
+    rev = "ef85b1393aeaec905f6310abca29b4e0f42db901";
```
Comment (matthewbauer) [inline pkgs/games/openmw/default.nix]: This should use the openmw-0.44.0 tag which corresponds to that hash for clarity.

**Comment by peterhoeg** [comment]
OpenMW has also had support for qt5 for the longest time - let's take this opportunity to move to that instead of qt4.

File: pkgs/games/openmw/default.nix (hunk 1 above)
Comment (hcmensch) [reply pkgs/games/openmw/default.nix]: whoops, I forgot about this. I've got to review packaging + github docs I think. I'm going to refrain from going near master until I get a bit more familiar, sorry about that & thanks for the the review tip.

**Comment by hcmensch** [comment]
@peterhoeg Forgot about this account sorry... & thanks for the comment. I'm probably capable of those changes after reviewing a bunch of stuff, but I am considering refraining from touching master until I am more familiar with a large code base like this (just a hobbyist/student now).

Would you mind advising me on good practices re: closing the PR or maybe pointing me to pertinent project guidelines if you can? Thanks for your time

**Comment by peterhoeg** [comment]
> I am considering refraining from touching master

You are not able to merge directly to master - if you open a PR we will review it before it gets merged, so you do not have to worry about that.

> pointing me to pertinent project guidelines

If you have a look at the 2 commits I referenced above, you can see how qt4 -> qt5 could be done.

We need all the contributors we can, so if you feel that you're up to it, please give it a go and feel free to ping me for input.

**Comment by ryantm** [comment]
@GrahamcOfBorg build openmw

**Pull request closed without merging**